pkg/scd/uss_availability_handler.go

```diff
@@ -1,12 +1,15 @@
 package scd
 
+import "log"
 import "context"
 import "github.com/interuss/dss/pkg/api/v1/scdpb"
 
 func (a *Server) GetUssAvailability(ctx context.Context, request *scdpb.GetUssAvailabilityRequest) (*scdpb.UssAvailabilityStatusResponse, error) {
-	panic("implement me")
+	log.Println("implement me")
+	return nil, nil
```

Comment:
I believe this will cause the endpoint to return a success code -- we should make sure these endpoints still indicate some kind of error even if they no longer take down the entire server when they are hit.  To accomplish this, we can look at other places where that happens and determine that we should return some kind of error.  Probably returning `stacktrace.NewError("Not implemented")` should work.